service/src/main/kotlin/xyz/lebkuchenfm/domain/radiopersonality/llmprompts/LlmSituationPromptsRepository.kt

```diff
@@ -0,0 +1,5 @@
+package xyz.lebkuchenfm.domain.radiopersonality.llmprompts
+
+interface LlmSituationPromptsRepository {
+    suspend fun findLatestByType(type: LlmSituationType): LlmSituationPrompt?
```

Comment:
🤔 maybe we should allow multiple active situational prompts? 
that way Radio Presenter could  randomly "greet me" / "complain about the taxi driver" / "express happiness cause of holidays" etc instead of mixing all the above in one message

Reply:
This should be expressed via personality prompts. Here we just need a description of what's happening - currently "song is changing" or "listener is calling". The whole sentiment of "you like to complain about the taxi drivers", "you love xmas" should be a part of personality prompt.